python-threatexchange/threatexchange/api_models.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+from datetime import datetime
+from dateutil.parser import parse
+from dataclasses import dataclass
+
+"""
+Models (dataclasses only) for interfacing with the threatexchange API.
+"""
+
+
+def _parse_datetime_from_iso_8601(datestr: str) -> datetime:
+    """
+    Parses strings representing date like 2019-05-20T16:44:47+0000 from the
+    graph api into datetime objects.
+    """
+    return parse(datestr)
+
+
+@dataclass
+class ThreatPrivacyGroup:
```

Comment:
Check out the pytx library (the original pytx, which I stole the name of for pytx3) for more ideas: https://github.com/facebook/ThreatExchange/blob/master/api-reference-examples/python/pytx/pytx/threat_privacy_group.py